unity_buy_android/unity_buy_plugin/src/main/java/com/shopify/unity/buy/AndroidPayCheckoutSession.java

```diff
@@ -0,0 +1,102 @@
+package com.shopify.unity.buy;
+
+import android.util.Log;
+
+import com.shopify.buy3.pay.PayCart;
+import com.unity3d.player.UnityPlayer;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.math.BigDecimal;
+
+public class AndroidPayCheckoutSession {
```

Comment:
nit: usually I make by default everything final (just to indicate that class wasn't designed for extension, and only when it need to be extended then think carefully how to do it right, as shared state is an evil)